turbopack/crates/turbopack-core/src/chunk/mod.rs

```diff
@@ -176,12 +175,13 @@ pub trait OutputChunk: Asset {
 pub enum ChunkingType {
     /// Module is placed in the same chunk group and is loaded in parallel. It
     /// doesn't become an async module when the referenced module is async.
```

Comment:
is this comment stale given the new field

Reply:
👍 🙈